arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -0,0 +1,1712 @@
+					"format": "date-time",
+					"type": "string"
+				},
+				"type": {
+					"enum": ["unknown", "export", "import", "backup", "readDeviceProperties", "writeDeviceProperties", "updateDeviceConfiguration", "rebootDevice", "factoryResetDevice", "firmwareUpdate"],
+					"type": "string"
+				},
+				"status": {
+					"enum": ["unknown", "enqueued", "running", "completed", "failed", "cancelled"],
+					"type": "string"
+				},
+				"failureReason": {
+					"type": "string"
+				},
+				"statusMessage": {
+					"type": "string"
+				},
+				"deviceId": {
+					"type": "string"
+				},
+				"parentJobId": {
+					"type": "string"
+				}
+			}
+		},
+		"ResponseWithContinuation[IEnumerable[IotHubQuotaMetricInfo]]": {
+			"type": "object",
+			"properties": {
+				"value": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/IotHubQuotaMetricInfo"
+					}
+				},
+				"nextLink": {
+					"type": "string",
+					"readOnly": true
+				}
+			}
+		},
+		"IotHubQuotaMetricInfo": {
+			"type": "object",
+			"properties": {
+				"Name": {
+					"type": "string"
+				},
+				"CurrentValue": {
+					"format": "int64",
+					"type": "integer"
+				},
+				"MaxValue": {
+					"format": "int64",
+					"type": "integer"
+				}
+			}
+		},
+		"OperationInputs": {
+			"type": "object",
+			"properties": {
+				"Name": {
+					"type": "string"
+				},
+				"ResourceType": {
+					"type": "string"
+				},
+				"Key": {
+					"type": "string"
+				},
+				"TokenLifeTimeInMinutes": {
+					"format": "int32",
+					"type": "integer"
+				},
+				"ConnectionString": {
+					"type": "string"
+				},
+				"IotHubDescription": {
+					"$ref": "#/definitions/IotHubDescription"
+				}
+			}
+		},
+		"IotHubNameAvailabilityInfo": {
+			"type": "object",
+			"properties": {
+				"nameAvailable": {
+					"type": "boolean"
+				},
+				"reason": {
+					"enum": ["Invalid", "AlreadyExists"],
+					"type": "string"
+				},
+				"message": {
+					"type": "string"
+				}
+			}
+		},
+		"ResponseWithContinuation[IEnumerable[SharedAccessSignatureAuthorizationRule]]": {
+			"type": "object",
+			"properties": {
+				"value": {
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/SharedAccessSignatureAuthorizationRule"
+					}
+				},
+				"nextLink": {
+					"type": "string",
+					"readOnly": true
+				}
+			}
+		},
+		"ExportDevicesRequest": {
+			"type": "object",
+			"properties": {
+				"ExportBlobContainerUri": {
+					"type": "string"
+				},
+				"ExcludeKeys": {
+					"type": "boolean"
+				}
+			}
+		},
+		"ImportDevicesRequest": {
+			"type": "object",
+			"properties": {
+				"InputBlobContainerUri": {
+					"type": "string"
+				},
+				"OutputBlobContainerUri": {
+					"type": "string"
+				}
+			}
+		}
+	},
+	"parameters": {
+		"api-version": {
+			"name": "api-version",
+			"in": "query",
+			"description": "Client API version.",
+			"required": true,
+			"type": "string",
+			"default": "2016-02-03"
+		},
+		"asyncinfo": {
```

Comment:
nope.. this one is required only for querying the status of the long running operation. If that's coming for free I can remove this altogether